packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -82,6 +119,7 @@ export const OnRemove: TagGroupStory = {
 };
 
 export const Wrapping: TagGroupStory = {
+  args: {width: '100%'},
```

Comment:
I'm worried about needing this 'width' prop on everything
can we include it in the component somehow?

and I think I just realized why, this is because fields are default 192px isn't it?
could we put the width 100% inside our component though and pass that to Field? then if a prop with a specific width is passed in, we pass that through to Field instead?

Or, should all TagGroups be 192 width... unless someone specifies the width like you have?

Reply:
This is just to demonstrate the wrapping behavior in storybook. These stories have a resizable container decorator, and this just ensures that it fills that container.

Reply:
but it should just fill the resizable container on its own? Like ActionGroup does https://reactspectrum.blob.core.windows.net/reactspectrum/59f432c581d1ca0da5ae71b1436c0a41f87f8156/storybook/index.html?path=/story/actiongroup--overflow

Reply:
You're right they do fill. Now that `width: auto` on the field is set, they no longer need width: '100%' to fill.

Reply:
The 'maxRows + onRemove' story still won't fill unless it has `width: '100%'`. Looking into this.